debezium-core/src/main/java/io/debezium/pipeline/spi/StreamingResult.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline.spi;
+
+public class StreamingResult<O extends OffsetContext> {
```

Comment:
A comment would be good. Let's try and have class-level comments in general.

Reply:
Noted.